mailscanner/detail.php

```diff
@@ -505,7 +509,11 @@
                 (defined('DOMAINADMIN_CAN_RELEASE_DANGEROUS_CONTENTS') && true === DOMAINADMIN_CAN_RELEASE_DANGEROUS_CONTENTS && $_SESSION['user_type'] === 'D') ||
                 $item['dangerous'] !== 'Y'
             ) {
-                echo '  <td align="center"><input type="checkbox" name="release[]" value="' . $item['id'] . '"></td>' . "\n";
+                if ($row['released'] > 0 && $item['file'] === 'message') {
+                    echo '  <td align="center" class="released"><input type="checkbox" name="release[]" value="' . $item['id'] . '"></td>' . "\n";
+                } else {
+                    echo '  <td align="center"><input type="checkbox" name="release[]" value="' . $item['id'] . '"></td>' . "\n";
+                }
```

Comment:
Could you refactor this to avoid duplication? just set the class name in a var (or empty string) and use the var in class="$var"